art-galleries-nyc/main.js

```diff
@@ -19,7 +19,6 @@ require([
   "esri/symbols/TextSymbol3DLayer",
 
   "esri/widgets/Search",
-  "dojo/domReady!"
```

Comment:
Many of these `dojo/` things remaining. They look like memories from an old past and are not necessary anymore. With 4.26 we cleaned up a bit in the API and thus these files are not served anymore. In all the cases in this PR, they were not actually needed for the sample to work. So... less code - yay!